src/App.js

```diff
@@ -1,23 +1,13 @@
-import logo from './logo.svg';
 import './App.css';
+import Main from './views/Main';
 
 function App() {
   return (
     <div className="App">
       <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
+        <h1> Countries of the World </h1>
       </header>
```

Comment:
Just an FYI, we will usually add in the <h1> into the header component to make the app congruent, as we did with <Main /> :)